LoopKit/DeviceManager/DeviceLog/StoredDeviceLogEntry.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  StoredDeviceLogEntry.swift
+//  LoopKit
+//
+//  Created by Pete Schwamb on 1/14/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+public struct StoredDeviceLogEntry {
+    public let type: DeviceLogEntryType
+    public let deviceManager: String
+    public let deviceIdentifier: String?
+    public let message: String
+    public let timestamp: Date
+    
+    public init(
+        type: DeviceLogEntryType,
+        deviceManager: String,
+        deviceIdentifier: String?,
+        message: String,
+        timestamp: Date
+    ) {
+        self.type = type
+        self.deviceManager = deviceManager
+        self.deviceIdentifier = deviceIdentifier
+        self.message = message
+        self.timestamp = timestamp
+    }
+
+    
+    init(managedObject: DeviceLogEntry) {
+        self.init(
+            type: managedObject.type!,
+            deviceManager: managedObject.managerIdentifier!,
```

Comment:
[nit] why not align the naming here between `deviceManager` and `managerIdentifier` like the other properties?